lib/commands.js

```diff
@@ -211,19 +208,26 @@ const scroll = async (direction = "down", amount = 300, method = "mouse") => {
       break;
     case "left":
       config.TD_VM
-        ? console.log("Not Supported")
+        ? console.error("Not Supported")
         : await robot.scrollMouse(amount, 0);
       await redraw.wait(2500);
       break;
     case "right":
       config.TD_VM
-        ? console.log("Not Supported")
+        ? console.error("Not Supported")
         : await robot.scrollMouse(amount * -1, 0);
       await redraw.wait(2500);
       break;
     default:
       throw new AiError("Direction not found");
   }
+  const after = await captureScreenBase64();
+
+  if (before === after) {
```

Comment:
![image](https://github.com/user-attachments/assets/aef8d368-a01f-4e3d-b558-67e81a0e26fc)

Reply:
I think this is appropriate given that we're only issuing a warning. Great DX dude!